pyrule_compendium/__init__.py

```diff
@@ -16,12 +16,14 @@ class compendium(object):
         * `default_timeout`: Default seconds to wait for response for all API calling functions until raising `requests.exceptions.ReadTimeout`
             - default: `None` (no timeout)
             - type: integer, float, tuple (for connect and read timeouts)
-            - notes: If an API calling function has a parameter `timeout`, it will overide this
+            - notes: If an API calling function has a parameter `timeout`, it will override this
     """
 
-    def __init__(self, base_url: str="https://botw-compendium.herokuapp.com/api/v2", default_timeout: Union[int, float, None]=None):
+    def __init__(self, base_url: str="https://botw-compendium.herokuapp.com/api/v2", default_timeout: Union[int, float, None]=None, master_mode: bool = False):
```

Comment:
I think `master_mode` should be passed into the functions rather than to the class, because we do not want to create another object for `master_mode` data.

Reply:
I feel that having `master_mode` passed when the class is initialized would remove the need for a redundant parameter in every function of the class.